osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -90,6 +95,11 @@ private double computeAimValue()
 
             aimValue *= lengthBonus;
 
+            if (mods.Any(h => h is OsuModRelax))
+            {
+                aimValue *= 0.6;
```

Comment:
Wouldn't it be easier to just adjust global multiplier instead of multiplying every skill separately?